package.json

```diff
@@ -7,15 +7,20 @@
     "typecheck": "tsc"
   },
   "dependencies": {
+    "@unlock-protocol/contracts": "0.0.22",
+    "@unlock-protocol/networks": "0.0.18",
```

Comment:
A simple package that includes the network configs (RPC endpoints). can be replaced with your own :)